code/api/src/migrations/6-style-user.js

```diff
@@ -0,0 +1,12 @@
+
+module.exports = {
+    up: (queryInterface, Sequelize) => {
+      return queryInterface.addColumn('users', 'style', {
+          type: Sequelize.STRING
+        });
+    },
+    down: (queryInterface, Sequelize) => {
+      return queryInterface.removeColumn('users', 'style');
+    }
+  }
```

Comment:
Looks good. I think style is fine as a string for now, we'll see if the FE prefers enums